src/components/Gallery/Gallery.scss

```diff
@@ -87,6 +88,7 @@ $verticalGalleryMargin: 28px;
         #{$block} {
             &__content {
                 width: 100vw;
+                min-width: 100vw;
```

Comment:
Is it make a difference? We specify the same value in `width`

Reply:
It's override of the `min-width: 480px` for the cases when the mobile screen is less than 480px